main.tf

```
@@ -2653,6 +2653,80 @@ module "vpa" {
   tags = var.tags
 }
 
+
+################################################################################
+# Fargate Fluentbit
+################################################################################
+# Help on Fargate Logging with Fluentbit and CloudWatch
+# https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html
+locals {
+  fargate_fluentbit_cwlog_group         = "/${var.cluster_name}/fargate-fluentbit-logs"
+  fargate_fluentbit_cwlog_stream_prefix = "fargate-logs-"
```

Lets try to be consistent in how we are asking users to provide configuration values for addons. This should look something like...

```suggestion
  ffargate_fluentbit_cwlog_group        = try(fargate_fluentbit.cwlog_group, "/${var.cluster_name}/fargate-fluentbit-logs")
  fargate_fluentbit_cwlog_stream_prefix = try(fargate_fluentbit.cwlog_stream_prefix "fargate-logs-")
```